tests/csrf_tests/tests.py

```diff
@@ -189,6 +189,17 @@ def test_csrf_token_in_header(self):
         req2 = CsrfViewMiddleware().process_view(req, post_form_view, (), {})
         self.assertIsNone(req2)
 
+    def test_csrf_token_in_header_with_customized_name(self):
+        """
+        Check that we can pass in the token in a header with customized name
+        instead of in the form
+        """
+        with self.settings(CSRF_HEADER_NAME="HTTP_X_CSRFTOKEN_CUSTOMIZED"):
```

Comment:
I'd prefer to decorator the test with `@override_settings` so we don't need to indent the entire test.